Missing `tslib` in production

The `netlify-plugin-next-dynamic` is missing the `tslib` dependency in production:

```
Could not import plugin:
Cannot find module 'tslib'
Require stack:
- /opt/build/repo/.netlify/plugins/node_modules/netlify-plugin-next-dynamic/index.js
- /opt/buildhome/.netlify-build-nvm/versions/node/v12.16.3/lib/node_modules/@netlify/build/src/plugins/child/logic.js
- /opt/buildhome/.netlify-build-nvm/versions/node/v12.16.3/lib/node_modules/@netlify/build/src/plugins/child/load.js
- /opt/buildhome/.netlify-build-nvm/versions/node/v12.16.3/lib/node_modules/@netlify/build/src/plugins/child/main.js 
    internal/modules/cjs/loader.js:957:15 Function.Module._resolveFilename
    internal/modules/cjs/loader.js:840:27 Function.Module._load
    internal/modules/cjs/loader.js:1019:19 Module.require
    internal/modules/cjs/helpers.js:77:18 require
    /opt/build/repo/.netlify/plugins/node_modules/netlify-plugin-next-dynamic/index.js:4:17 Object.<anonymous>
    internal/modules/cjs/loader.js:1133:30 Module._compile
    internal/modules/cjs/loader.js:1153:10 Object.Module._extensions..js
    internal/modules/cjs/loader.js:977:32 Module.load
    internal/modules/cjs/loader.js:877:14 Function.Module._load
    internal/modules/cjs/loader.js:1019:19 Module.require
```

If you check the transpiled code of `index.js`:

```js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.onPostBuild = void 0;
const tslib_1 = require("tslib");
const generateRedirects_1 = require("./functions/generateRedirects");
const path = require("path");
exports.onPostBuild = (opts) => tslib_1.__awaiter(void 0, void 0, void 0, function* () {
    const { constants, inputs } = opts;
    const { nextDir = path.join(constants.PUBLISH_DIR, '../.next') } = inputs;
    yield generateRedirects_1.generateRedirects(opts)(nextDir, constants.PUBLISH_DIR);
});
//# sourceMappingURL=index.js.map
```

You see that `tslib` is required to transpile `async`/`await`. To make it work, I believe you would need to either:
  - Require `tslib` as a production dependency
  - Use the `target` TypeScript option in order to inline `tslib` helpers

--- a/libs/netlify-plugin-next-dynamic/package.json
+++ b/libs/netlify-plugin-next-dynamic/package.json
@@ -2,7 +2,7 @@
   "name": "netlify-plugin-next-dynamic",
   "description": "Netlify plugin that allows you to deploy dynamic NextJS path statically",
   "author": "Phumrapee Limpianchop <contact@rayriffy.com>",
-  "version": "1.0.8",
+  "version": "1.0.9",
   "homepage": "https://github.com/Brikl/opensource/tree/master/libs/netlify-plugin-next-dynamic#readme",
   "bugs": {
     "url": "https://github.com/Brikl/opensource/issues"

--- a/tsconfig.base.json
+++ b/tsconfig.base.json
@@ -7,7 +7,7 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "importHelpers": true,
+    "importHelpers": false,
     "target": "es2015",
     "module": "esnext",
     "typeRoots": ["node_modules/@types"],

